entrypoint.sh

```diff
@@ -43,6 +44,7 @@ aws() {
     [[ "${AWS_SUBNET_ID}" ]] && optionals+="-subnetid ${AWS_SUBNET_ID} " # if not set retrieved from IMDS
     [[ "${AWS_REGION}" ]] && optionals+="-aws-region ${AWS_REGION} "     # if not set retrieved from IMDS
     [[ "${TAGS}" ]] && optionals+="-tags ${TAGS} " # Custom tags applied to pod vm
+    [[ "${USE_PUBLIC_IP}" ]] && optionals+="-use-public-ip " # Use public IP for pod vm
```

Comment:
I've got two comments for this change otherwise looks great:

1. Here compare with "true" (`[[ "${USE_PUBLIC_IP}" = "true" ]]`) as `USE_PUBLIC_IP` can be either `false` or exported empty too.
2. Any reason to not include the new property in `install/overlays/aws/kustomization.yaml`?

Reply:
@wainersm I have updated the PR to incorporate the changes. I missed adding the new property in the overlays yaml